config/default/manager_custom_env.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
```

Comment:
This is a file generated by make update-mgr-env and does not need to be committed.